app/views/talk/show.html.erb

```diff
@@ -1,2 +1,2 @@
-<div><p style="margin-top: 10px;font-family: 'Junction Light';clear:left;"><%= raw t('talk.show.welcome', :page => @slug, :url1 => '/wiki/'+@slug, :url2 => '/wiki/talk-pages') %></p></div>
-<div><iframe src="https://pad.publiclab.org/p/<%= @slug[0,50] %>" style="width: 100%; height:800px;"><p><%= t('talk.show.try_direct_link', :url1 => 'https://pad.publiclab.org/p/'+@slug[0,50]) %></p></iframe></div>
+<div><p style="margin-top: 10px;font-family: 'Junction Light';clear:left;"><%= raw t('talk.show.welcome', :page => @node.latest.title, :url1 => '/wiki/'+@node.latest.title, :url2 => '/wiki/talk-pages') %></p></div>
```

Comment:
Sorry -- `path` actually already includes the string "/wiki/".  So the first instance of `@node.latest.title` is perfect, but for the second you could actually replace all of `'/wiki/'+@node.latest.title` with just `@node.path`

Then on the line below, both instances of `@node.path[0,50]` should be `@node.slug[0,50]` because we want just the part /after/ the "/wiki/" string -- I know this is a little confusing, but thanks!